File: src/main/java/com/example/desafio/api/models/repositorys/BrandRepository.java
package com.example.desafio.api.models.repositorys;

import com.example.desafio.api.models.entitys.BrandEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BrandRepository extends JpaRepository<BrandEntity, Long> {
    public BrandEntity findByNameBrand(String nameBrand);
}

File: src/main/kotlin/com/example/desafio/api/models/repositorys/BrandRepository.kt
package com.example.desafio.api.models.repositorys

import com.example.desafio.api.models.entitys.BrandEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BrandRepository : JpaRepository<BrandEntity?, Long?> {
    fun findByNameBrand(nameBrand: String?): BrandEntity?
}